common/models/user.js

```diff
@@ -42,11 +42,15 @@ var debug = require('debug')('loopback:user');
  * - ALLOW OWNER `findById`
  * - ALLOW OWNER `updateAttributes`
  *
- * @property {String} username Must be unique
- * @property {String} password Hidden from remote clients
- * @property {String} email Must be valid email
- * @property {Boolean} emailVerified Set when a user's email has been verified via `confirm()`
- * @property {String} verificationToken Set when `verify()` is called
+ * @property {String} username Must be unique.
+ * @property {String} password Hidden from remote clients.
+ * @property {String} email Must be valid email.
+ * @property {Boolean} emailVerified Set when a user's email has been verified via `confirm()`.
+ * @property {String} verificationToken Set when `verify()` is called.
+ * @property {String} realm Specific domain a user belongs to, [Learn more](https://docs.strongloop.com/display/public/LB/Partitioning+users+with+realms).
```

Comment:
@bajtos  Not too familiar with the realm concept, can you take a look to see if this make sense?